### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by jaureguif**

### CODE REVIEW

File: src/main/java/com/epam/asset/tracking/api/EntityController.java [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+			throw new DuplicateKeyExceptionWrapper("Duplicated username: " + bp.getUsername(), dke);
+			
+		}
+
+	}
+
+}
+
+class Pojo {
+
```
Comment (dan-zx) [inline src/main/java/com/epam/asset/tracking/api/EntityController.java:138]: Is this class still in use?

File: src/main/java/com/epam/asset/tracking/dto/validations/ValidationsUtil.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
+package com.epam.asset.tracking.dto.validations;
+
+public interface ValidationsUtil {
+
```
Comment (dan-zx) [inline src/main/java/com/epam/asset/tracking/dto/validations/ValidationsUtil.java]: Is there a reason for this class to be an interface?

**Review by dan-zx — CHANGES_REQUESTED** [review]

**Comment by dan-zx** [comment]
We can use later this lib to help us with orika mapper configuarion: [orika-spring-boot-starter](https://github.com/akihyro/orika-spring-boot-starter)

File: src/main/java/com/epam/asset/tracking/api/EntityController.java (hunk 1 above)
Comment (jaureguif) [reply src/main/java/com/epam/asset/tracking/api/EntityController.java:138]: For now, yes

**Pull request merged**